api-tests/.golangci-required.yml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
It seems that we can remove this file.

Reply:
Are u sure? In this repo it is unusued, but it is used in makefile, it can be maybe used in some jenkins pipeline